lib/hooks/useTime.ts

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react';
+import { formatter } from '../utils';
+
+const useTime = () => {
+  const SunCalc = require('suncalc');
```

Comment:
Try moving this line to the top of the file and change the `require` syntax into standard `import` syntax.